recipes/itk/all/conanfile.py

```diff
@@ -48,34 +48,55 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
-
-        if self.settings.compiler == "apple-clang":
-            raise ConanInvalidConfiguration("itk does not support this compiler")
 
     def requirements(self):
         self.requires("libjpeg/9d")
-        self.requires("dcmtk/3.6.5")
+        self.requires("dcmtk/3.6.6")
         self.requires("double-conversion/3.1.5")
         self.requires("eigen/3.3.9")
-        self.requires("expat/2.2.10")
+        self.requires("expat/2.4.1")
         self.requires("fftw/3.3.9")
+        self.requires("gdcm/3.0.9")
         self.requires("hdf5/1.12.0")
-        self.requires("icu/68.2")
+        self.requires("icu/69.1")
         self.requires("libtiff/4.2.0")
         self.requires("libpng/1.6.37")
         self.requires("openjpeg/2.4.0")
+        self.requires("tbb/2020.3")
         self.requires("zlib/1.2.11")
 
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "14",
+            "gcc": "4.8.1",
```

Comment:
The version compare is pessimistic

```suggestion
            "gcc": "4.8",
```

Reply:
I've taken it from here: https://gcc.gnu.org/projects/cxx-status.html
"GCC 4.8.1 was the first feature-complete implementation of the 2011 C++ standard, previously known as C++0x."

Reply:
Yes but the Conan tool will block the default settings 4.8 ... the compilers version should match the default setting (so it works for the most consumers) but it's just a note and difficult to enforce... we also have our "lazy semver" which is used by some recipes where this is important